MaxText/integration/tunix/tunix_adaptor.py

```diff
@@ -0,0 +1,49 @@
+from __future__ import annotations
+
+from typing import Optional, Tuple, Any
+
+import jax
+import jax.numpy as jnp
+from flax import nnx
+
+from MaxText.common_types import MODEL_MODE_TRAIN, MODEL_MODE_AUTOREGRESSIVE, Config
+from MaxText.layers.models import Transformer
+
+Array = jax.Array  # Alias for clarity
```

Comment:
from jax import Array is probably more idiomatic?

Reply:
done